mlflow/transformers.py

```diff
@@ -217,7 +217,7 @@ def save_model(
     processor=None,
     task: Optional[str] = None,
     model_card=None,
-    inference_config: Optional[Dict[str, Any]] = None,
+    model_config: Optional[Dict[str, Any]] = None,
```

Comment:
As discussed, could we revert this change and keep inference_config for now?
We should add extra load_config that's consistent with pyfunc's save_model/log_model.

Reply:
There are some dependencies on our side with `inference_config` utilization with our inference platform. We should avoid modifying this for `transformers` :) Maybe we'll make that breaking change with 3.0

Reply:
@BenWilson2, any dependency on existing platforms should be transparently handled because the change is backward compatible. If you want users to continue using the old way we can mark it as deprecated instead of removing from signature and doing the breaking change in 3.0. But I don't think we should keep both. It will be just a confusing signature for users with two options to achieve the same. What do you think?

Reply:
yes, let's deprecate `inference_config` and introduce `model_config`. When users specify `inference_config`, we can print a warning. @BenWilson2 @serena-ruan can you communicate this to the Databricks Model Serving team?

Reply:
Will do!